input/otlp/traces.go

```diff
@@ -175,6 +175,10 @@ func (c *Consumer) convertSpan(
 			Sampled:             true,
 			RepresentativeCount: representativeCount,
 		}
+		event.Span = &modelpb.Span{
```

Comment:
**issue**: you need to check that `spanID` is not empty before assigning, otherwise the event output will encode an empty `event.Span` and the tests will fail. (`event.Span` would be no longer nil)

Reply:
this was fixed in d6249c0983c1e8ee75fd7123cba858dedc3fe269